server/controllers/userController.js

```diff
@@ -0,0 +1,29 @@
+import bcrypt from 'bcryptjs';
+import models from '../database/models';
+import helpers from '../helpers';
+
+const { responseMessage, createToken } = helpers;
+const { User } = models;
+const saltRounds = 10;
+
```

Comment:
You need to include JSDoc comments for the UserController class

Reply:
done